src/user/views/author_views.py

```diff
@@ -753,14 +762,12 @@ def add_publications(self, request, pk=None):
             # if True:
             if TESTING:
                 pull_openalex_author_works_batch(openalex_ids, request.user.id)
+                invalidate_author_profile_caches(author.id)
             else:
-                pull_openalex_author_works_batch.apply_async(
-                    (
-                        openalex_ids,
-                        request.user.id,
-                    ),
-                    priority=1,
-                )
+                chain(
+                    pull_openalex_author_works_batch.s(openalex_ids, request.user.id),
+                    invalidate_author_profile_caches.s(author.id),
```

Comment:
This guarantees`invalidate_author_profile_caches` is fired after `pull_openalex_author_works_batch.` finishes?